profile/README.md

```diff
@@ -12,20 +12,19 @@ data.
 
 <br clear="right"/>
 
-# Arcaflow
-
+# [Arcaflow](https://arcalot.io/arcaflow)
 <a href="https://arcalot.io/arcaflow"><img align="left" width="200px"
 alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various
 plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.svg"></a>
 
-Arcaflow is a workflow system consisting of three main components:
+Arcaflow is a workflow orchestration system consisting of three main components:
 
-* [The Arcaflow engine](https://github.com/arcalot/arcaflow-engine), statically-compiled
-and platform-portable, written in Go
-* [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-), written in
-various languages and delivered as Linux containers or modules
-* [Workflow definitions](https://github.com/arcalot/arcaflow-workflows), written in
-YAML, tying together plugins and their data
+* [The Arcaflow engine](https://github.com/arcalot/arcaflow-engine) - A
+platform-independent binary, written in Go
+* [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-) - Delivered
+as Linux containers and developed with SDKs
```

Comment:
Pragmatically, yes, we expect plugins would be developed with an SDK (it'd be difficult to write and more difficult to maintain without that layer). But I'm not sure it's really a useful phrase at this level. You're running a *container*, so why would a new user care?

Reply:
Each of these very short tl;dr's has a user and a developer aspect to it. I don't think these need to be pedantically accurate as the audience is someone who is reading the word Arcaflow for the very first time.

Reply:
> a user and a developer aspect to it

Then they should probably be more clearly separated to avoid everyone being confused by the diffuse focus. A casual user doesn't care about SDKs. I got you to back off on the "any language" *but* now you're going the other direction by implying that an SDK is *required*. Though I'd agree that writing a plugin without an SDK would be a bit foolhardy, this reversal seems overly strong.